pgen-native/Makefile

```diff
@@ -0,0 +1,25 @@
+CFLAGS := -Wall -g
+
+OBJS := parser.o \
+	tokenizer.o \
+	grammar1.o \
+	grammar.o \
+        bitset.o \
+        node.o \
+	python.o \
+	acceler.o \
+        listnode.o \
+	python_grammar.o \
+	parsetok.o \
+	main.o
+
+.PHONY: parser
```

Comment:
Dependency tracking is a royal pain to set up in make, so I just made the main target .PHONY to force a rebuild every time. It really doesn't take long on modern hardware...